tests/driver.c

```diff
@@ -138,16 +138,30 @@ void test_find_transactions_obj_by_tag(void) {
   printf("Average time of find_tx_obj_by_tag: %lf\n", sum / TEST_COUNT);
 }
 
+void test_send_mam_message(void) {
+  double sum = 0;
+  status_t ret = SC_OK;
+  bool last_packet = false;
+  for (size_t count = 0; count < TEST_COUNT; count++) {
+    test_time_start(&start_time);
+    ret = api_mam_send_message(&ta_core.tangle, &ta_core.service, TEST_PAYLOAD,
+                               last_packet);
+    TEST_ASSERT_EQUAL_INT32(ret, SC_OK);
+    test_time_end(&start_time, &end_time, &sum);
```

Comment:
This should executed before `TEST_ASSERT`

Reply:
But in this way we could know the error with error number
Is it better to use `TEST_ASSERT`